src/config/config.ts

```diff
@@ -17,6 +17,11 @@ export const CONFIG = {
         description:
           'Path to the output IMPL file where the results as saved, if none is provided it prints to stdout.',
       },
+      overrideParams: {
```

Comment:
please check if it works with 'override-params'

Reply:
works, thanks